client/src/controllers/RevealController.test.js

```diff
@@ -455,4 +455,69 @@ describe('RevealController', () => {
       expect(document.querySelector('header').outerHTML).toEqual(previousHTML);
     });
   });
+
+  describe('save the expanded state of breadcrumbs to local storage', () => {
+    beforeEach(async () => {
+      jest.spyOn(Storage.prototype, 'setItem').mockImplementation(() => {});
+      jest.spyOn(Storage.prototype, 'getItem').mockImplementation(() => {});
+    });
+
+    afterEach(() => {
+      jest.restoreAllMocks();
+    });
+
```

Comment:
For the tests below, let's add some tests for asserting the local storage gets called. I can add this when merging though.